{"name": "test_login", "status": "failed", "statusDetails": {"message": "AssertionError: assert 5 == 6", "trace": "self = <test_case_my_page.TestMapage object at 0x0000015E6CA29748>\n\n    @allure.feature(\"登录4测试\")\n    def test_login(self):\n        # self.my_page.login_password(\"13000000000\", \"abc123456\")\n        # assert self.my_page.get_attr_value(\"text\") != \"登录/注册\"\n>       assert 5==6\nE       assert 5 == 6\n\ntest_case_my_page.py:24: AssertionError"}, "start": 1678065839691, "stop": 1678065839691, "uuid": "58cd32c0-775e-4a4e-8eb7-11c6c456c901", "historyId": "5c74f42c0d1368dc21939ba9d783b7a1", "testCaseId": "0207d3df1cdd0cb05eb2b97cf522d9ed", "fullName": "test_case_my_page.TestMapage#test_login", "labels": [{"name": "feature", "value": "登录4测试"}, {"name": "suite", "value": "test_case_my_page"}, {"name": "subSuite", "value": "TestMapage"}, {"name": "host", "value": "PC-20200423TYSC"}, {"name": "thread", "value": "97892-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case_my_page"}]}